docs/source/development/traffic_router/traffic_router_api.rst

```diff
@@ -471,3 +471,65 @@ Request Structure
+
+Request Structure
+-----------------
+.. table:: Request Query Parameters
+
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| Name              | Required | Description                                                                                                  |
+	+===================+==========+==============================================================================================================+
+	| requestPath       | yes      | The (URI encoded) request path to use to test pattern based consistent hashing                               |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| deliveryServiceId | yes      | The integral, unique identifier?/'xml_id'?/name? of a :term:`Delivery Service` served by this Traffic Router |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+
+Response Structure
+------------------
+TBD
+
+``/crs/consistenthash/cache/coveragezone/steering``
+===================================================
+The resulting cache of the consistent hash using coverage zone for a given client IP, delivery service and, request path -- used to test cache selection for steering delivery services.
+
+Request Structure
+-----------------
+.. table:: Request Query Parameters
+
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| Name              | Required | Description                                                                                                  |
+	+===================+==========+==============================================================================================================+
+	| requestPath       | yes      | The (URI encoded) request path to use to test pattern based consistent hashing                               |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| deliveryServiceId | yes      | The integral, unique identifier?/'xml_id'?/name? of a :term:`Delivery Service` served by this Traffic Router |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| ip                | yes      | The IP address of a potential client                                                                         |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+
+Response Structure
+------------------
+TBD
```

Comment:
"TBD" is not an acceptable response structure. It should minimally have an example - though the only reason any of these don't have field explanations is because I don't understand them (yet), and since you know exactly what they are what they mean, you should document it so others can know too.

Reply:
accidental duplicate :P